Sources/CodeEditTextView/Extensions/STTextView+/STTextView+TextInterface.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  STTextView+TextInterface.swift
+//  
+//
+//  Created by Khan Winter on 1/26/23.
+//
+
+import AppKit
+import STTextView
+import TextStory
+import TextFormation
+
+extension STTextView: TextInterface {
+    public var selectedRange: NSRange {
+        get {
+            return self.selectedRange()
+        }
+        set {
+            if let textRange = NSTextRange(newValue, provider: textContentStorage) {
+                self.setSelectedRange(textRange)
+            }
+        }
+    }
```

Comment:
Small thing, but wouldn't it be clearer if the `self.selectedRange()` method was renamed to `self.setSelectedRange()`? I was confused, because at first sight it looks like the computed var calls itself.

Reply:
That variable is part of the conformance to `TextInterface` so it can't be changed sadly.

Reply:
I believe that's implemented by STTextView so it's outside our implementation. Let me check

Reply:
Yeah that can't be changed.

Reply:
> Yeah that can't be changed.

Oh okay, my bad!